.pipelines/deployment/arc-extension-release/ServiceGroupRoot/buildver.txt

```diff
@@ -0,0 +1 @@
+0.0.1
```

Comment:
Hi Grace, can you tell us where we use this ver file and when we increment it ?

Reply:
Thanks Vishwa, here is the ev2 docs on it: https://ev2docs.azure.net/features/authoring/version/version.html. In our other ev2 files and for ContainerInsights, we have actually never used it since we have the image tag build id. However, I can change it so that we are putting the same as what we have in our VERSION file here?

Reply:
thanks .